models/db/search.go

```diff
@@ -18,12 +18,6 @@ const (
 	SearchOrderByRecentUpdated         SearchOrderBy = "updated_unix DESC"
 	SearchOrderByOldest                SearchOrderBy = "created_unix ASC"
 	SearchOrderByNewest                SearchOrderBy = "created_unix DESC"
-	SearchOrderBySize                  SearchOrderBy = "size ASC"
```

Comment:
Why not keep them here?

Reply:
because they are not generic but table "repository" specific

Reply:
of course if there would exist an other table witch we want to order by column 'size' it should belong in generig db search ... but I did not found one - has that changed or is something planed?